dev-install-locally-no-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./gradlew clean install -x test
```

Comment:
I'll test it out. Install _should_ build it as well, shouldn't it? Will do

On Fri, Apr 8, 2016 at 07:19, Misagh Moayyed <notifications@github.com> wrote:
In dev-install-locally-no-tests.sh [https://github.com/Jasig/cas/pull/1676#discussion_r59010972] :

> @@ -0,0 +1,3 @@
> +#!/usr/bin/env bash
> +
> +./gradlew clean install -x test


You could likely turn the existing script into:

./gradlew clean install -DskipAspectJ=true -x javadoc -x test


Would that work?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/Jasig/cas/pull/1676/files/9c8a4b49aee43d91c0113ec6f61045218cf9e71f#r59010972]